### PR TITLE
add online resources

### PR DESCRIPTION
**Pull request by MissDove**
Resources added:
egghead.io
Frontend Masters
LinkedIn Learning

### CODE REVIEW

File: online-resources/learn-to-code-resources.md [hunk 1]
```diff
@@ -31,6 +31,7 @@
 * [DeVry Bootcamp](https://www.devry.edu/)
 
 * [EdX](https://www.edx.org/)
+* [Egghead.io](https://egghead.io/)
```
Comment (charlottebrf) [inline online-resources/learn-to-code-resources.md]: Yes! Good idea

File: online-resources/learn-to-code-resources.md [hunk 2]
```diff
@@ -39,10 +40,12 @@
 * [Famous.co](https://famous.co/)
 * [Free Code Camp](https://learn.freecodecamp.org/)
 * [Free online university courses](https://medium.freecodecamp.org/200-universities-just-launched-560-free-online-courses-heres-the-full-list-d9dd13600b04)
+* [Frontend Masters](https://frontendmasters.com/)
```
Comment (charlottebrf) [inline online-resources/learn-to-code-resources.md]: Never seen this before, nice one 👍

File: online-resources/learn-to-code-resources.md [hunk 3]
```diff
@@ -39,10 +40,12 @@
...
 
 * [Khan Academy - Computing](https://www.khanacademy.org/computing)
 
 * [Learn to code](https://aartik1.github.io/index.html)
+* [LinkedIn Learning](https://www.linkedin.com/learning/) // The same content as on Lynda.com, but you get Premium LinkedIn account with it
```
Comment (charlottebrf) [inline online-resources/learn-to-code-resources.md]: Aha! Didn't know that thanks

File: online-resources/learn-to-code-resources.md (hunk 1 above)
Comment (MissDove) [reply online-resources/learn-to-code-resources.md]: 😉

File: online-resources/learn-to-code-resources.md (hunk 2 above)
Comment (MissDove) [reply online-resources/learn-to-code-resources.md]: Yes, it's really good, especially for really understanding JavaScript. Kyle Simpson (You Don't Know JS series) has many videos there 😄

File: online-resources/learn-to-code-resources.md (hunk 3 above)
Comment (MissDove) [reply online-resources/learn-to-code-resources.md]: Yes, someone else told me this too 😉

**Pull request merged**